src/openrct2/network/Network.cpp

```diff
@@ -31,7 +31,7 @@
 // This string specifies which version of network stream current build uses.
 // It is used for making sure only compatible builds get connected, even within
 // single OpenRCT2 version.
-#define NETWORK_STREAM_VERSION "21"
+#define NETWORK_STREAM_VERSION "22"
```

Comment:
Network version was already 22

Reply:
This has been addressed by https://github.com/OpenRCT2/OpenRCT2/commit/2368fea91e17fe8b51788fcbd68401f74874d918.

Reply:
How did this actually happen, looking at the diff it was 21 actually

Reply:
It was due to when you rebased. It doesn't show a merge clash if you are making the exact same change so it didn't throw up a warning.